components/NewsletterModal.js

```diff
@@ -0,0 +1,160 @@
+import React, { useContext, useState } from 'react'
+import styled, { ThemeContext } from 'styled-components'
+import axios from 'axios';
+import Button from './Button'
+import Modal from './Modal'
+import { Body, Title2 } from './Typography'
+
+export default function NewsletterModal({ show, onClose }) {
+  const themeContext = useContext(ThemeContext);
+  const [inputMessage, setInputMessage] = useState('');
+
+  const ComboButton = styled(Button)`
+    margin-top: -41px;
+    margin-left: calc(100% - 200px);
+
+    ${themeContext.mediaQueries.mobile} {
+      height: 36px;
+      width: 90px;
+      margin-top: -36px;
+      margin-left: calc(100% - 90px);
+    }
+
+    &:hover {
+      cursor: pointer;
+    }
+  `;
+
+  const Content = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  `;
+
+  const Description = styled.div`
+    width: 50%;
+
+    ${themeContext.mediaQueries.mobile} {
+      width: 75%;
+    }
+  `;
+
+  const Logo = styled.img`
+    ${themeContext.mediaQueries.mobile} {
+      width: 60px;
+    }
+  `;
+
+  const StyledInput = styled.input`
+    border: 2px solid ${themeContext.colors.primary};
+    border-radius: 8px;
+    padding: 8px 12px;
+    background: transparent;
+    color: white;
+    font-weight: normal;
+    font-size: 18px;
+    width: 100%;
+
+    ${themeContext.mediaQueries.mobile} {
+      padding: 4px 8px;
+      font-size: 14px;
+      line-height: 24px;
+    }
+  `;
+
+  const emailInput = React.createRef();
+  function addToMailingList() {
+    // Reset any error/info messages from before
+    setInputMessage('');
+    
+    const email = emailInput.current.value;
+    const validEmail = validateEmail(email);
+
+    if (validEmail) {
+      axios({
+        method: 'POST',
+        url: 'https://us-central1-nwplus-ubc.cloudfunctions.net/addToMailingList',
+        data: {
+          email: email,
+        },
+      }).then((res) => {
+        console.log(res);
```

Comment:
think you missed removing this console log hehe

Reply:
thank you king, removed in 43f5958